city_scrapers/spiders/il_aging_advisory_council.py

```diff
@@ -0,0 +1,113 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlAgingAdvisoryCouncilSpider(CityScrapersSpider):
+    name = "il_aging_advisory_council"
+    agency = "Illinois Aging Advisory Council"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www2.illinois.gov/aging/PartnersProviders/OlderAdult/Pages/acmeetings.aspx"
+    ]
+    location = {
```

Comment:
`location` should only return a `dict` with string keys for `address` and `name`. Some of the Illinois ones have been a bit odd because they'll have multiple locations, but we've been using the Chicago one when this happens

Reply:
This has been updated to return `location` with only Chicago address and name